### ISSUE TITLE
Government flavors

### ISSUE THREAD

**Issue by rquinio** [loup99/WtWSMS#142]
Issue to track addition of government flavors recently added in vanilla.
It's just an event-like image and a flavor name.

**Comment by rquinio** [comment]
Some things I've noticed:
- WRE / ERE bureaucratic government defaults to Anarchy, which is kind of bad.
- Roman successor kingdom / states could get their own flavor names.

**Comment by EmperorEnlil** [comment]
I could work on this

**Comment by TurtleShroom** [comment]
Unfortunately, I don't have the artistic skill to make CK2-quality banners, but I can still give you some PNG or TGA files to look at as placeholders. I have software that can convert DDS to PNG and JPG, but not PNG and JPG to DDS.

**Comment by TurtleShroom** [comment]
Hello! I made a placeholder for the Bureaucracy government flavor image. I know an actual artist who could attempt to draw a real placeholder,. but for now, use this:

![bureaucracy](https://cloud.githubusercontent.com/assets/17663470/14391134/49abd1a8-fd88-11e5-8961-4147c7ad9dae.png)

**Comment by TurtleShroom** [comment]
What do you think?

**Comment by rquinio** [comment]
That's cute! :) 
Though I think we must stay consistent in the style of pictures with vanilla.
By the way the text is not hardcoded in the picture, but added from localization (and seems to allow localization commands)

**Comment by TurtleShroom** [comment]
I don't think we should have to stick to the original game style. The flags I gave you for the Root West files, which you accepted, were not consistent with the game design. I made them using MS Paint and Art Cursors.

I never suggested using my particular image as a permanent feature. I suggessted using it _until_ you can get a better artist to make a real image.
